perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -198,8 +221,29 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
+    self.ConfigureVMKernel()
     self.BurnCpu()
 
+  def ConfigureVMKernel(self):
+    """Apply {procfs,sysfs}_config to VM."""
+
+    path_components = None
+
+    def ConfigureSubtree(subtree):
+      for name, value in subtree.iteritems():
+        path_components.append(name)
+        if not isinstance(value, dict):
+          self.RemoteCommand('echo "%s" | sudo tee %s' %
+                             (str(value), posixpath.join(*path_components)))
```

Comment:
What happens if that path does not exist?  Ie "echo foo > /proc/foo/bar.txt" where /proc/foo/ does not exist as a directory?  I'm not sure if this is even a case -- if the user passes in a value to set that the OS does not know about (the "foo" in this case) should we error out?